docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md

```diff
@@ -85,6 +85,12 @@ export EKSA_VSPHERE_USERNAME='<vSphere-username>'
 export EKSA_VSPHERE_PASSWORD='<vSphere-password>'
 ```
 
+### error unmarshaling JSON: while decoding JSON: json: unknown field "spec"
+```
+Error: loading config file "cluster.yaml": error unmarshaling JSON: while decoding JSON: json: unknown field "spec"
+```
+Use `eksctl anywhere create cluster -f cluster.yaml` instead of `eksctl create cluster -f cluster.yaml`
```

Comment:
```suggestion
Use `eksctl anywhere create cluster -f cluster.yaml` instead of `eksctl create cluster -f cluster.yaml` to create an EKS Anywhere cluster.
```